src/org/usfirst/frc/team2335/robot/Robot.java

```diff
@@ -81,6 +90,11 @@ public void teleopInit()
 	@Override
 	public void teleopPeriodic()
 	{
+		yVal = oi.getAxis(Y_AXIS, 1);
```

Comment:
Why do you create xVal and yVal outside of teleop but define them inside?

Reply:
Because allocating new memory for each iteration of teleopPeriodic could create some garbage (allocated, unused memory).